### PR TITLE
COL-1229 Zoom controls for Activity Network

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-1229

Do these make the tool more usable, or less? The viewers will decide!

### CODE REVIEW

File: public/app/dashboard/activityNetworkDirective.js [hunk 1]
```diff
@@ -329,22 +341,53 @@
...
+            var aspectRatio = viewportWidth * 1.0 / viewportHeight;
+
+            svg.attr('width', viewportWidth).attr('height', viewportHeight);
+            clipPathRect.attr('width', viewportWidth).attr('height', viewportHeight);
+
+            var xMargin = Math.max(0, (Math.round(activityNetworkScale * aspectRatio) - viewportWidth) / 2);
+            var yMargin = Math.max(0, (activityNetworkScale - viewportHeight) / 2);
+            bounds = [[-1 * xMargin, -1 * yMargin], [viewportWidth + xMargin, viewportHeight + yMargin]];
+            scope.minScale = Math.min(viewportHeight / activityNetworkScale, 1);
+            zoom.scaleExtent([scope.minScale, 1]).traånslateExtent(bounds);
```
Comment (johncrossman) [inline public/app/dashboard/activityNetworkDirective.js]: @paulkerschen Please tell me that 'traånslateExtent' is correct...

**Comment by johncrossman** [comment]
I approve, @paulkerschen.  Posted one question.

File: public/app/dashboard/activityNetworkDirective.js (hunk 1 above)
Comment (paulkerschen) [reply public/app/dashboard/activityNetworkDirective.js]: Hvorfor gjør Sublime Text disse tegnene???

**Comment by johncrossman** [comment]
Perfekt. Fint arbeid!

**Pull request merged**